release-note.txt

```diff
@@ -0,0 +1,17 @@
+Release v0.6.1 release notes.
+
+New features:
+    The 'paictl' tool: Introducing paictl, the universal deployment/management tool with the functionalities of image building, service start/stop, k8s bootup/clean, and configuration generation.
+    Single-box deployment: Support deploying on a single machine.
```

Comment:
>Support deploying on a single machine [](start = 27, length = 37)

support single-box deployment for evaluation purpose.

Reply:
Done